Automattic/jetpack
# renovate: Use configured version of PHP when running composer

#### Changes proposed in this Pull Request:

In the absence of configuration or certain dependencies, renovate
currently defaults to running composer with PHP 7.4. Or, more
specifically, it defaults to whatever is the 'latest' tag on the
`renovate/php` docker package.

We do everything else with PHP 8.0, so configure renovate to use that
too.

**:warning: If we merge this, WooCommerce will probably stop working in the Jetpack Development Environment with errors like the one [here](https://href.li?https://github.com/woocommerce/woocommerce/issues/33611#issuecomment-1170280202).**

#### Other information:


#### Jetpack product discussion


None, but see https://github.com/Automattic/jetpack/pull/25106#issuecomment-1185611334

#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:






* Check that it's not too close to the scheduled renovate run.
* Go to [the dependency dashboard](https://href.li?https://github.com/Automattic/jetpack/issues/23363) and check the box for "Lock file maintenance".
* Go to [the action page](https://github.com/Automattic/jetpack/actions/workflows/renovate.yml) and trigger a dry run, specifying this branch.
* Wait for the triggered run to finish.
* Go to [the dependency dashboard](https://href.li?https://github.com/Automattic/jetpack/issues/23363) and uncheck the box for "Lock file maintenance".
* Check the logs for indications that it's using PHP 8.0 rather than 7.4
  * :heavy_check_mark: [Using a docker image `docker.io/renovate/php:8.0.21`](https://github.com/Automattic/jetpack/runs/7363125518?check_suite_focus=true#step:5:29956)
  * :heavy_check_mark: [Updating symfony/string to 6.0.x, not 5.x (PHP 7.4) or 6.1.x (PHP 8.1)](https://github.com/Automattic/jetpack/runs/7363125518?check_suite_focus=true#step:5:29963)
